### PR TITLE
Feature/update floating controlbar margin

### PR DESCRIPTION
**Pull request by madikarizma**
Inset control bars no longer float on small player size.

Fixes #
JW7-3247

### CODE REVIEW

**Comment by madikarizma** [comment]
Please note that max-width removal was reverted, despite it not showing in "changes": https://github.com/jwplayer/jwplayer/blob/feature/update-floating-controlbar-margin/src/css/imports/mixins.less#L31

File: src/css/imports/controlbar.less [hunk 1]
```diff
@@ -14,13 +14,22 @@
     }
 }
 
+.jw-breakpoint-0 .jw-controlbar {
```
Comment (robwalch) [inline src/css/imports/controlbar.less]: Another problem with `.inset-controlbar()` is that it sets `display: inline-block;` when the controlbar should not be displayed while the player is idle.

```css
.inset-controlbar() {
    .jw-controlbar {
        // for correct display in IE browsers
        display: inline-block;
        // width assignment is required for IE11 to center correctly
        width: 96%;
        max-width: 50em;
        margin: 0 auto;
        bottom: .7em;
        left: 2%;
        right: 2%;
    }
    &.jw-flag-audio-player .jw-controlbar {
        bottom: 0; // overwrite the inset-controlbar mixin
    }
}
```

File: src/css/imports/mixins.less [hunk 2]
```diff
@@ -37,6 +37,14 @@
     &.jw-flag-audio-player .jw-controlbar {
         bottom: 0; // overwrite the inset-controlbar mixin
     }
+    &.jw-breakpoint-0 .jw-controlbar {
```
Comment (madikarizma) [inline src/css/imports/mixins.less]: @robwalch Should be better now

Comment (robwalch) [inline src/css/imports/mixins.less]: http://player-pr-test-jenkins.longtailvideo.com/builds/3805/archive/test/squash/bootstrap.html?primary=html5&edition=ads&feature=skins_premium_skins

```js
for (var i = 0; i < 100; i++) {
    var jw = jwplayer(i);
    if (typeof jw.resize !== 'function') {
        break;
    }
    jw.resize('100%', 300);
}
```

File: src/css/imports/mixins.less [hunk 3]
```diff
@@ -41,7 +41,6 @@
         border-radius: 0;
         bottom: 0;
         left: 0;
-        padding: 0;
```
Comment (madikarizma) [inline src/css/imports/mixins.less]: @robwalch

**Comment by madikarizma** [comment]
@robwalch adjustments here https://github.com/jwplayer/jwplayer/pull/1517/commits/205df5e1658d62a5fed58813a734c96297b8165e

**Pull request merged**